src/test/resources/complex2.javadoc

```diff
@@ -0,0 +1,50 @@
+ * This class demonstrates a full-scale Javadoc parsing stress test.
+ *
+ * <h4>🚨 Extreme Cases in One Comment 🚨</h4>
```

Comment:
I couldn't remember if we had input with emojis or not, so I wanted to add this as well.